pkg/apis/devices/v1beta1/device_instance_types.go

```diff
@@ -263,9 +269,8 @@ type VisitorConfig struct {
 type Device struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-
-	Spec   DeviceSpec   `json:"spec,omitempty"`
-	Status DeviceStatus `json:"status,omitempty"`
+	Spec              DeviceSpec   `json:"spec,omitempty"`
+	Status            DeviceStatus `json:"status,omitempty"`
```

Comment:
```suggestion
	Spec    DeviceSpec   `json:"spec,omitempty"`
	Status  DeviceStatus `json:"status,omitempty"`
```
Why need to insert a lot of spaces?